prac_07/car.py

```diff
@@ -4,23 +4,21 @@
 class Car:
     """Represent a Car object."""
 
-    def __init__(self, name='', fuel=0):
+    def __init__(self, fuel=0):
```

Comment:
Did you copy the old car, instead of using your new car? It's OK, but I would have used the one you had improved.